hid-lg4ff.c

```diff
@@ -248,6 +253,9 @@ static const struct lg4ff_multimode_wheel lg4ff_multimode_wheels[] = {
 	{USB_DEVICE_ID_LOGITECH_G29_WHEEL,
 	 LG4FF_MODE_NATIVE | LG4FF_MODE_G29 | LG4FF_MODE_G27 | LG4FF_MODE_G25 | LG4FF_MODE_DFGT | LG4FF_MODE_DFP | LG4FF_MODE_DFEX,
 	 LG4FF_G29_TAG, LG4FF_G29_NAME},
+	{USB_DEVICE_ID_LOGITECH_G923_WHEEL,
+	 LG4FF_MODE_NATIVE | LG4FF_MODE_G923 | LG4FF_MODE_G29,
```

Comment:
Here you should add the other modes supported by the G29.

Reply:
Added in new commit